src/moduler/utskrift/print/aktiviteter.tsx

```diff
@@ -103,9 +103,14 @@ function Aktiviteter(props: Props) {
         return null;
     }
 
-    const gruperteAktiviteter = aktiviteter.reduce((map, a) => {
-        map[a.status] ? map[a.status].push(a) : (map[a.status] = [a]);
-        return map;
+    const gruperteAktiviteter = aktiviteter.reduce((acc: { [key: string]: Aktivitet[] | undefined }, a: Aktivitet) => {
```

Comment:
Skjønner ikke denne helt men tenker det samme som over. Kan det ikke her returneres tom array isteden for undefined når det ikke er resultat

Reply:
min typescript foo er ikke bra nokk for dette